Sorts/test/MergeSort.test.ts

```diff
@@ -0,0 +1,20 @@
+import { MergeSort } from "../MergeSort";
+
+/**
+ * Space complexity - O(n)
```

Comment:
This belongs in the JSDoc comment of the merge sort func?

Reply:
Shifted the complexity part to the MergeSort.ts file